cloudflair.py

```diff
@@ -58,14 +60,24 @@ def find_hosts(domain, censys_api_id, censys_api_secret, use_cloudfront):
 
     print('[*] Looking for certificates matching "%s" using Censys' % domain)
     cert_fingerprints = censys_search.get_certificates(domain, censys_api_id, censys_api_secret)
-    print('[*] %d certificates matching "%s" found.' % (len(cert_fingerprints), domain) )
+    cert_fingerprints = list(cert_fingerprints)
+    cert_fingerprints_count = len(cert_fingerprints)
+    print('[*] %d certificates matching "%s" found.' % (cert_fingerprints_count, domain))
 
-    if len(cert_fingerprints) == 0:
+    if cert_fingerprints_count == 0:
         print('Exiting.')
         exit(0)
 
+    chunking = (cert_fingerprints_count > 25)
```

Comment:
is there a reason why this is 25 and not something else? We should also probably extract it in a variable since it's used in several other places below. That's my only comment on the PR, nice job!

Reply:
Its mainly bc we have to template them into Censys Search Queries, which do have a maximum length. So with the longer sha256 hashes, I broke them down into 25s. I can totally make it a constant up top!